### ISSUE TITLE
RevitPythonShell add-in don't appear in Revit

### ISSUE THREAD

**Issue by Mairh93** [architecture-building-systems/revitpythonshell#117]
Hello all, 

I am using the Revit Version 2021.1 and I downloaded the Revitpythonshell.addin for Revit 2021. I copy paste the .addin file in this path "C:\Users\name..\AppData\Roaming\Autodesk\Revit\Addins\2021" as it was come in the admin folder. 

The problem is that when I open Revit I get the error below : 
![revitshell](https://user-images.githubusercontent.com/27021809/130597569-02c85998-bb53-48e3-9d20-785e318b0f6a.jpg)

Do you have any idea? Thank you in advance!

**Comment by daren-thomas** [comment]
Hmm... this was all a _very_ long time ago, but it looks like you might not have copied everything you need to. Check the admin folder where you got the `.addin` file from: maybe there is a similar folder that is called "RevitPythonShell"? You could try copying that too. Also, check the contents of the `.addin` file. Maybe that contains some information?

**Comment by Mairh93** [comment]
Yes, I just found that I should also copy the folder ( C:\Users\...admin\AppData\Roaming\**RevitPythonShell**) in the user folder. Thank you for your time!
